Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -3141,15 +3145,25 @@ bool AccessibilityObject::supportsARIAAttributes() const
 }
     
 AccessibilityObject* AccessibilityObject::elementAccessibilityHitTest(const IntPoint& point) const
-{ 
+{
     // Send the hit test back into the sub-frame if necessary.
     if (isAttachment()) {
         Widget* widget = widgetForAttachmentView();
         // Normalize the point for the widget's bounds.
         if (widget && widget->isLocalFrameView()) {
-            if (AXObjectCache* cache = axObjectCache())
+            if (CheckedPtr cache = axObjectCache())
                 return cache->getOrCreate(*widget)->accessibilityHitTest(IntPoint(point - widget->frameRect().location()));
         }
+
+        if (widget && widget->isRemoteFrameView()) {
+            if (CheckedPtr cache = axObjectCache()) {
+                if (RefPtr remoteHostWidget = cache->getOrCreate(*widget)) {
+                    remoteHostWidget->updateChildrenIfNecessary();
+                    RefPtr scrollView = dynamicDowncast<AccessibilityScrollView>(remoteHostWidget);
```

Comment:
Adding a `*` to this cast will save a null-check, i.e. `dynamicDowncast<...>(*remoteHostWidget)`